modules/olmv1-catalog-queries.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * operators/olm_v1/olmv1-installing-an-operator-from-a-catalog.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="olmv1-catalog-queries_{context}"]
+= Common catalog queries
+
+You can query catalogs by using the `jq` CLI tool.
+
+// https://docs.asciidoctor.org/asciidoc/latest/tables/format-cell-content/#a-operator
+// https://docs.asciidoctor.org/asciidoc/latest/tables/data-format/#escape-the-cell-separator
+.Common catalog queries
```

Comment:
Would it make sense to have subsections for common queries against specific FBC blob types? Maybe something like:

- Common queries
    - Common Package queries
    - Common Channel queries
    - Common Bundle queries

Reply:
@everettraven Thanks, Bryce! I took a stab at the changes. Would you please add the comment you made in the refinement meeting about the organization? Apologies for having a hard time keeping track during the meeting.